### PR TITLE
Calculate quantiles when building NumberSummary

### PR DESCRIPTION
**Pull request by cswarth**
```
QuantileSummary qs = colProfile.toColumnSummary().getNumberSummary().getQuantiles();
System.out.printf("JSON Quantiles %s\n", JsonFormat.printer().print(qs));
```

before:
```
JSON Quantiles {}
```

after:
```
JSON Quantiles {
  "quantiles": [0.0, 0.01, 0.05, 0.25, 0.5, 0.75, 0.95, 0.99, 1.0],
  "quantileValues": [-2354.131591796875, -868.7899780273438, -445.5729064941406, 211.17071533203125, 669.4403076171875, 1136.6209716796875, 1845.1197509765625, 2449.074462890625, 4070.94970703125]
}
```

### CODE REVIEW

**Comment by cswarth** [comment]
@andyndang not sure if this is the way you want to go, but it mimics what the python client is doing.
I'd be interested if you have better way to do the type conversions needed to build the 
QuantileSummary.

**Comment by cswarth** [comment]
@andyndang I'll do it because spotless insists, but this style change is terrible.

from:
```
    for (int index = 0; index < qvals.length; index++)
      boxedQvals[index] = (double) (qvals[index]);
```

to:
```
    for (int index = 0; index < qvals.length; index++) boxedQvals[index] = (double) (qvals[index]);
```

**Comment by cswarth** [comment]
Failing tests that seem to have nothing to do with my change,

```
Run starting. Expected test count is: 7
Using Spark's default log4j profile: org/apache/spark/log4j-defaults.properties
21/04/09 20:07:59 WARN Utils: Service 'sparkDriver' could not bind on a random free port. You may check whether configuring an appropriate binding address.
[...]
21/04/09 20:07:59 ERROR SparkContext: Error initializing SparkContext.
java.net.BindException: Cannot assign requested address: Service 'sparkDriver' failed after 16 retries (on a random free port)! Consider explicitly setting the appropriate binding address for the service 'sparkDriver' (for example spark.driver.bindAddress for SparkDriver) to the correct binding address.
```

**Comment by andyndang** [comment]
> @andyndang I'll do it because spotless insists, but this style change is terrible.
> 
> from:
> 
> ```
>     for (int index = 0; index < qvals.length; index++)
>       boxedQvals[index] = (double) (qvals[index]);
> ```
> 
> to:
> 
> ```
>     for (int index = 0; index < qvals.length; index++) boxedQvals[index] = (double) (qvals[index]);
> ```

You should wrap them in curly brackets `{}`.

File: core/src/main/java/com/whylogs/core/SummaryConverters.java [hunk 1]
```diff
@@ -124,6 +128,21 @@ public static NumberSummary fromNumberTracker(NumberTracker numberTracker) {
...
 
+    // some unfortunate type mismatches requires conversions.
+    val QUANTILES = new double[] {0.0, 0.01, 0.05, 0.25, 0.5, 0.75, 0.95, 0.99, 1.0};
+    val boxedQuantiles = toObject(QUANTILES);
+
+    // unfortunately ArrayUtils does not have a static utility for converting float[] -> double[].
+    val qvals = numberTracker.getHistogram().getQuantiles(QUANTILES);
+    val len = qvals.length;
+    val boxedQvals = new Double[len];
+    for (int index = 0; index < qvals.length; index++) boxedQvals[index] = (double) (qvals[index]);
```
Comment (andyndang) [inline core/src/main/java/com/whylogs/core/SummaryConverters.java]:
```suggestion
    for (int index = 0; index < qvals.length; index++) {
        boxedQvals[index] = (double) (qvals[index]);
    }
```

**Review by andyndang — DISMISSED** [review]
LGTM. I added a minor style suggestion

**Pull request merged**